src/__tests__/fixtures/providers/cognito-auth.provider.ts

```diff
@@ -0,0 +1,32 @@
+import {Provider} from '@loopback/context';
+import {Request} from '@loopback/rest';
+import {Cognito, IAuthUser, VerifyFunction} from '../../../types';
+
+/**
+ * A provider for default implementation of VerifyFunction.CognitoAuthFn
+ *
+ * It will just throw an error saying Not Implemented
```

Comment:
Doesn't look like it will do this 🙂

Reply:
This is mock-verify-provider naa. It will just return the user (like other verify-providers).
This comment is from the actual verify provider, forgot to remove it. 😅

Reply:
ok 🙁